src/sage/knots/link.py

```diff
@@ -3104,9 +3101,6 @@ def is_colorable(self, n):
 
         - ``n`` -- the number of colors to consider
```

Comment:
I would add the same comment as before:

```
 - ``n`` -- the number of colors to consider (if ommitted the
          value of the determinant of ``self`` will be taken
```

Reply:
Thanks! I've done it in the current commit.